Add links to tokenizer API docs to refer relevant information. (#4064)
explosion/spaCy: website/docs/api/tokenizer.md

source: spacy/tokenizer.pyx
---

> Segment text, and create `Doc` objects with the discovered segment boundaries. For a deeper understanding read about [how spaCy's tokenizer works](/usage/linguistic-features#how-tokenizer-works).

Reviewer: ```suggestion
Segment text, and create `Doc` objects with the discovered segment boundaries. For a deeper understanding, see the docs on [how spaCy's tokenizer works](/usage/linguistic-features#how-tokenizer-works).
```
